services/notifications.go

```diff
@@ -731,6 +726,13 @@ func queueEmailNotifications(notificationsByUserID map[uint64]map[types.EventNam
 
 					metrics.NotificationsQueued.WithLabelValues("email", string(event)).Inc()
 				}
+
+				if len(notificationTitles) > 2 {
+					subject = fmt.Sprintf("%s: %s,... and %d other notifications", utils.Config.Frontend.SiteDomain, notificationTitles[0], len(notificationTitles)-1)
+				} else {
+					subject = fmt.Sprintf("%s: %s and %s", utils.Config.Frontend.SiteDomain, notificationTitles[0], notificationTitles[1])
+				}
```

Comment:
This code does not work if there is only one notification title, does it?

Also, it should not be part of the `for` loop it is currently part of, right? It should be done after the `for`.

Reply:
Right, fixed.

Also fixed. I thought it already was outside the loop.